Corrected numbering issue on- How to Install and Configure Playground… (#159)
ironprogrammer/test-handbook: get-setup-for-testing/test-core-tickets-with-playground.md

```
<img src="https://make.wordpress.org/test/files/2026/04/playground-cli-in-terminal-scaled.webp" alt="Run PlayGround CLI in Terminal" style="max-width: 100%">

> <p>

Reviewer: Do we need p tags for markdown?

Author: No, Should remove it? and keep only text?

Reviewer: We don't need `p` tags here as well as the `code` tags. We must use Markdown as much as possible. They will get converted to HTML when PR is merged.

Please check the existing handbook pages, you will see that we never used `p` or `code` tags. Instead we use backticks for inline code and triple backticks for codeblock.

I use this VSCode Extension for editing and previewing Markdown files:
https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced

<img width="1083" height="761" alt="md-preview" src="https://github.com/user-attachments/assets/e7a248b8-e02c-4c14-a6ff-d584ecc85618" />

Your WordPress instance should now be accessible at: <a href="http://localhost:9400">http://localhost:9400</a>
<br />
<img src="https://make.wordpress.org/test/files/2026/04/playground-cli-in-browser-scaled.webp" alt="WordPress Playground running locally on port 9400" style="max-width: 100%">
</p>

> <p>You may also consider creating an alias for this long command in your terminal, so that you don't need to remember and type it every time.</p>

Reviewer 2: ```suggestion
Your WordPress instance should now be accessible at: `http://localhost:9400`
<br />
<img src="https://make.wordpress.org/test/files/2026/04/playground-cli-in-browser-scaled.webp" alt="WordPress Playground running locally on port 9400" style="max-width: 100%">

You may also consider creating an alias for this long command in your terminal, so that you don't need to remember and type it every time.
```

Reviewer 3: The `a` tags are also not necessary; we can use MD links

[http://localhost:9400](http://localhost:9400)

Reviewer 2: @3kori Thanks for pointing that out. I’ve updated it.

> <p>Optionally, you can mount your themes folder if you will be adding code snippets into your active theme's functions.php file and want to keep the changes between rebuilds. To avoid affecting future tests, don't forget to remove any custom code snippets you have added.

Reviewer 2: ```suggestion
Optionally, you can mount your `themes` folder if you will be adding code snippets into your active theme's `functions.php` file and want to keep the changes between rebuilds. To avoid affecting future tests, don't forget to remove any custom code snippets you have added.
```

> <p>Optionally, you can mount your <code>themes</code> folder if you will be adding code snippets into your active theme's <code>functions.php</code> file and want to keep the changes between rebuilds. To avoid affecting future tests, don't forget to remove any custom code snippets you have added.

Reviewer: Why did you change the markdown backticks to code tags?

> Here is how you would mount the themes folder alongside the plugins.

Reviewer 2: ```suggestion
Here is how you would mount the `themes` folder alongside the `plugins`.
```

> Here is how you would mount the <code>themes</code> folder alongside the plugins.

Reviewer: Same here

> <p>You may also consider creating an alias for this long command in your terminal, so that you don't need to remember and type it every time.</p>

Reviewer: Sorry but you have removed the inline code that were between backticks.
Also `p` tags are still in the code.

Maybe you can consider starting with a brand new branch from the trunk and only fix the bullet points without changing the remaining markup.

  --login \
  --site-url=http://localhost:9400 \
  --mount=./src/wp-content/plugins:/wordpress/wp-content/plugins \
  --mount=./src/wp-content/themes:/wordpress/wp-content/themes
```

> </p>

Reviewer 2: ```suggestion
```
